addons/google_calendar/models/res_users.py

```diff
@@ -185,3 +185,17 @@ def check_synchronization_status(self):
                 sync_status = 'sync_stopped'
         res['google_calendar'] = sync_status
         return res
+
+    def _has_any_active_synchronization(self):
+        """
+        Check if synchronization is active for Google Calendar.
+        This function retrieves the synchronization status from the user's environment
+        and checks if the Google Calendar synchronization is active.
+
+        Returns:
+            bool: True if synchronization is active for Google Calendar, False otherwise.
```

Comment:
Hello, you should follow the same docstring pattern as you used above in this PR. Example:
```
:return: Action to delete the event
```